grafana-mixin/alerts/alerts.yaml

```diff
@@ -5,8 +5,8 @@ groups:
         for: 5m
         expr: |
           100 * namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query", status_code=~"5.."}
-          /
-          namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query"}
+          / ignoring (status_code)
+          sum without (status_code) (namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query"})
           > 0.5
```

Comment:
I'm unsure whether this was supposed to be >50% errors; the current >0.5% seems very low for most people. Should I make this configurable with jsonnet?

Reply:
That would be great :)

Reply:
I moved the rules and alerts into `.libsonnet` files. Let me know what you think.